go/tasks/plugins/k8s/kfoperators/mpi/mpi.go

```diff
@@ -67,6 +67,21 @@ func (mpiOperatorResourceHandler) BuildResource(ctx context.Context, taskCtx plu
 		return nil, flyteerr.Errorf(flyteerr.BadTaskSpecification, "Unable to create pod spec: [%v]", err.Error())
 	}
 
+	common.OverrideDefaultContainerName(taskCtx, podSpec, kubeflowv1.MPIJobDefaultContainerName)
```

Comment:
@hamersaw Do you think adding this line is incorrect?
As opposed to the pytorch and tfjob plugins, the container name is currently not overridden with `mpi` even though this is the default container name (see [here](https://github.com/kubeflow/training-operator/blob/ec53930b77e6c10adb75b966acb4bda13364ae8e/pkg/apis/kubeflow.org/v1/mpi_types.go#L28) and [here](https://github.com/kubeflow/training-operator/blob/ec53930b77e6c10adb75b966acb4bda13364ae8e/examples/mpi/tensorflow-mnist.yaml#L16)).

Reply:
@hamersaw I will try and have a look by the end of the week!